sdk/appconfiguration/app-configuration/test/public/utils/testHelpers.ts

```diff
@@ -54,20 +59,13 @@ export function getTokenAuthenticationCredential(): CredsAndEndpoint | undefined
   ];
 
   for (const name of requiredEnvironmentVariables) {
-    const value = env[name];
-
-    if (value == null) {
-      if (tokenCredentialsNotPresentWarning) {
-        tokenCredentialsNotPresentWarning = true;
-        console.log("Functional tests not running - set client identity variables to activate");
-      }
-
-      return undefined;
+    if (env[name] == null) {
+      throw new Error("Functional tests not running - set client identity variables to activate");
     }
   }
 
   return {
-    credential: new DefaultAzureCredential(),
+    credential: createTestCredential(options),
```

Comment:
How were we able to use DAC before, but were unable to use it in the implementation of `createTestCredential`?

Reply:
I believe the test was skipped.

![image](https://user-images.githubusercontent.com/97020683/147991464-faa30a3c-da04-4277-862f-6030c4e58edb.png)

Richard added that long ago probably with the hope to update to something more useful, but never got to it maybe.